chart/kubeapps/templates/kubeappsapis/configmap.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "kubeapps.kubeappsapis.fullname" . }}-configmap
+data:
+  plugins.conf: |-
+{{- if .Values.kubeappsapis.pluginConfig }}
+{{ .Values.kubeappsapis.pluginConfig | toPrettyJson | indent 4 }}
+{{- end }}
```

Comment:
I wonder if this config map should be conditional, rather than just the value here. Currently it means that if no pluginConfig is specified, we'll have this unused configmap being created with an empty value, whereas if the condition wrapped the whole config map, it'd simply not be created?